### ISSUE TITLE
Inconsistent entry with iPad "magic keyboard" in text input fields

### ISSUE THREAD

**Issue by psifertex** [emilk/egui#502]
Using the iPad magic keyboard I cannot input text immediately into the demo text fields. If I focus away and click around a few times or take some other actions (pasting appear to be one) it will suddenly become live and I can enter text.

On reload sometimes the keyboard works, sometimes it doesn't. 

Possibly related: using the "select all" hotkey (CMD-a) results in an extra cursor being visible (see screenshot)![image](https://user-images.githubusercontent.com/140215/122941725-a6d24600-d343-11eb-837b-8ac2ad30fa6c.jpeg)

**Comment by emilk** [comment]
Is this still an issue on www.egui.rs?

**Comment by rquinn2** [comment]
I don‘t think so. I am not able to recreate this on an IPad Pro 12.9 (2020) with magic keyboard running iPadOS 15.6.1.

That being said - there are numerous issues still, such as lack of copy/paste from egui.

**Comment by emilk** [comment]
> I don‘t think so. I am not able to recreate this on an IPad Pro 12.9 (2020) with magic keyboard running iPadOS 15.6.1.

Great, thanks for testing!


> That being said - there are numerous issues still, such as lack of copy/paste from egui.

There's another issue for this: https://github.com/emilk/egui/issues/501

**Comment by rquinn2** [comment]
My apologies @emilk  - I did some more testing on this as I‘ve been using egui a lot recently. I can indeed recreate both issues in the OP, The  inconsistent text entry is only apparent sometimes, it seems. The <Ctrl-a> adding a second cursor always happens.  

If I can get some time I‘ll try to look into what‘s causing it.
